code/game/machinery/iv_drip.dm

```diff
@@ -8,23 +8,36 @@
 	icon_state = "iv_drip"
 	anchored = FALSE
 	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
+	var/internal_can = FALSE
+	var/start_can = null
+	var/cantype = null
 	var/mob/living/carbon/attached = null
 	var/mode = IV_INJECTING
 	var/obj/item/reagent_containers/beaker = null
 	var/static/list/drip_containers = typecacheof(list(/obj/item/reagent_containers/blood,
 									/obj/item/reagent_containers/food,
 									/obj/item/reagent_containers/glass))
+/obj/machinery/iv_drip/saline
+	name = "Saline drip"
+	desc = "An all-you-can-drip saline canister designed to supply a hospital without running out, with a scary looking pump rigged to inject saline into containers, but filling people directly might be a bad idea."
+	icon_state = "saline"
+	density = TRUE
+	internal_can = /obj/item/reagent_containers/blood/saline
 
 /obj/machinery/iv_drip/Initialize()
```

Comment:
toss `mapload` as the first arg

Reply:
next I'm stupid and don't know where/what that means

Reply:
```suggestion
/obj/machinery/iv_drip/Initialize(mapload)
```